app/modules/assignment/reducers/assignment.js

```diff
@@ -9,7 +9,7 @@ import {
 } from "../constants"
 
 const initialState = {
-  name: null,
+  title: null,
```

Comment:
Switched this to `title` because the API response from Classroom now returns `title`(since that's what it's called on the Classroom side) instead of `name` and it was causing problems in the tests because of the two different naming conventions.